variables.tf

```diff
@@ -63,6 +63,14 @@ variable "require_imdsv2" {
   default = false
 }
 
+variable "imdsv2_hops" {
+  description = "How many hops to allow for IMDSv2 token requests. " +
+                "You might want to modify this if you're having problems " +
+                "with docker"
+  type = number
+  default = 2  # Allow for hops through one host to it's docker containers.
```

Comment:
The module will be a bit more general if the default value here is set to 1 or `null` (treated as missing) to be more in line with the actual default value from the AWS provider

Reply:
Agreed 💯